docs/guide-tequila-api-key.md

```diff
@@ -1,6 +1,20 @@
 ---
-title: "Guide: Get your Tequila API key"
-sidebar_label: "Guide: Get your Tequila API key"
+title: Guide: Get your Tequila API key
+sidebar_label: Guide: Get your Tequila API key
 ---
 
-_This page is a stub._
+For successful run our the GraphQL proxy server, you need to has set the Tequila **API key** to `API_KEY` environment variable.
+
+To get your **API key** you will need an account at the [Tequila](tequila.kiwi.com)
+. You can register and log in on the tequila.kiwi.com page.
+
+After successfully signing in to your account, you can find out what **Tequila** is about, how to work directly with the **Tequila API**.
```

Comment:
After successfully signing in to your account, you can read more information about **Tequila** and also explore the standard REST API.

Reply:
done